react/week3/todo/src/components/TodoItem.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+//typecheck
+import PropTypes from "prop-types";
+
+const TodoItem = (props) => {
+  const { todoId, todoDescription, todoDeadline } = props;
+  const [check, setCheck] = useState(false);
+  const [updatedTodo, setUpdatedTodo] = useState(todoDescription);
+  const [input, setInput] = useState(false);
+  const isDone = () => setCheck(!check);
```

Comment:
It is a convention to assign a boolean value to variables that are prefixed with `has` or `is`. Eg. `const isDone = false`